files/acl.py

```diff
@@ -21,6 +21,8 @@
 short_description: Sets and retrieves file ACL information.
 description:
      - Sets and retrieves file ACL information.
+notes:
+    - As of Ansible 2.0, this module only supports Linux distributions.
```

Comment:
Plus we are talking about a new major version for Ansible (therefore making non-BC changes if following semantic versioning).
When is Ansible 2.0 due? Chances are I'll send a further PR before it gets out.